custom/icds/templates/icds/messaging/indicators/kan/child_vaccinations_complete.txt

```diff
@@ -0,0 +1 @@
+Congratulations! You've given all the vaccines to {{ child_name }}
```

Comment:
Also this one.

Reply:
@millerdev This was left out intentionally because this message is not active. The Ministry removed this message from the scope of release so we don't intend to translate this message at this time.